tests/components/tuya/test_climate.py

```diff
@@ -55,3 +59,78 @@ async def test_platform_setup_no_discovery(
     assert not er.async_entries_for_config_entry(
         entity_registry, mock_config_entry.entry_id
     )
+
+
+@pytest.mark.parametrize(
+    "mock_device_code",
+    ["kt_serenelife_slpac905wuk_air_conditioner"],
+)
+async def test_fan_mode_windspeed(
+    hass: HomeAssistant,
+    mock_manager: ManagerCompat,
+    mock_config_entry: MockConfigEntry,
+    mock_device: CustomerDevice,
+) -> None:
+    """Test fan mode with windspeed."""
+    await initialize_entry(hass, mock_manager, mock_config_entry, mock_device)
+
+    state = hass.states.get("climate.air_conditioner")
+    assert state is not None, "climate.air_conditioner does not exist"
+    assert state.attributes["fan_mode"] == WINDSPEED_LOW
+    await hass.services.async_call(
+        Platform.CLIMATE,
+        "set_fan_mode",
+        {
+            "entity_id": "climate.air_conditioner",
+            "fan_mode": WINDSPEED_HIGH,
+        },
+    )
+    await hass.async_block_till_done()
```

Comment:
Ensure that send_commands has been called with correct arguments:
```suggestion
    await hass.async_block_till_done()
    mock_manager.send_commands.assert_called_once_with(
        mock_device.id, [{"code": "windspeed", "value": "2"}]
    )
```